.drone.yml

```diff
@@ -3,13 +3,42 @@ workspace:
   path: src/github.com/drone-plugins/drone-docker
 
 pipeline:
-  build:
-    image: golang:1.9
+  dep:
+    image: golang:1.11
     commands:
       - go get -u github.com/golang/dep/cmd/dep
       - dep ensure
-      - go vet ./...
+
+  fmt:
+    image: golang:1.11
+    group: validate
+    commands:
+      - GOFILES_NOVENDOR=$(find . -type f -name '*.go' -not -path "./vendor/*")
+      - gofmt -l -s -e $GOFILES_NOVENDOR
+      - exit $(gofmt -l -s -e $GOFILES_NOVENDOR | wc -l)
```

Comment:
Why do you add that?

Reply:
The `exit`? So that the number of lines is used as the error code.

Reply:
And what happens if it corrects more than 255 formats? :)

Reply:
But an exit code bigger than 255 is still invalid

Reply:
@tboerger Honestly, if one has 255 formatting errors in their code, a pipeline is the least of their problems.

Reply:
IMHO it's simply wrong to use the number of wrong formats as the exit code.

Reply:
The issue is that `gofmt` returns 0 regardless, found out that one can use `test -z` to return the right thing instead.